README.md

```diff
@@ -53,9 +70,214 @@ This is the simplest case. Get the script in whatever way you prefer and include
+
+The Brightcove Player [embed ID][bc-embed-id] for the player. The default value is correct for most users.
+
+#### `embedOptions`
+* *Type:* Object
+* *Default:* `undefined`
+
+Used to provide certain options for embed generation. These include:
+
+##### `embedOptions.pip`
+* *Type:* boolean
+
+If `true`, will wrap the embed in a `<div class="vjs-pip-container">` element. This should be used when you need support for [the Brightcove Picture-in-Picture plugin][bc-pip].
+
+##### `embedOptions.playlist`
+* *Type:* boolean
+
+If `true`, will add a `<div class="vjs-playlist">` element after the embed. This should be used when you need support for [the Brightcove Playlist UI plugin][bc-playlists].
+
+##### `embedOptions.responsive`
+* *Type:* boolean | Object
+
+> **NOTE:** This approach should work for in-page embeds, but it is recommended that you use the Video.js `fluid` and/or `aspectRatio` options instead:
+>
+> ```js
+> options: {
+>   aspectRatio: '16:9'
+> }
+> ```
+
+Used customize the embed code to produce a responsively-sized player using [the "intrinsic ratio" wrapper approach][bc-responsive-iframe].
+
+When `true`, will produce a responsive embed code with a 16:9 aspect ratio that will fill its container.
+
+An object can be provided to customize this with the following sub-properties:
+
+- `aspectRatio`: A string Used to customize the aspect ratio to a value other than 16:9 (e.g., `'4:3'`).
+- `maxWidth`: A string used to restrain the maximum width of the player. This should use CSS units, such as pixels (e.g., `'960px'`).
+
+#### `embedType`
+* *Type:* string
+* *Default:* `'in-page'`
+
+The [embed code type][bc-embed-code-type] to produce. This parameter's value must be one of the following:
+
+* `'in-page'` or `brightcovePlayerLoader.EMBED_TYPE_IN_PAGE`: Also referred to as the advanced embed code, this injects the player directly into the top-level web page.
+* `'iframe'` or `brightcovePlayerLoader.EMBED_TYPE_IFRAME`: Also referred to as the basic embed code, this injects the player as an `<iframe>` element.
+
+#### `onEmbedCreated`
+* *Type:* Function
+* *Default:* `undefined`
+
+A callback used to customize the embed element (either `video-js` element or an `iframe` element) *before* it is inserted into the DOM or customized as a result of `embedOptions` and before the player is downloaded and initialized.
+
+The return value of this function is ignored; the embed element should be mutated.
+
+Potential use-cases are adding/removing attributes or adding child elements, such as sources or tracks.
+
+#### `options`
+* *Type:* Object
+* *Default:* `undefined`
+
+A [Video.js options object][vjs-options] to pass during the player creation process. These options will take precedence over any settings specified in the Brightcove Player configuration.
+
+> **NOTE:** _Cannot be used with iframe embeds!_
+
+#### `playerId`
+* *Type:* string
+* *Default:* `'default'`
+
+A Brightcove Player ID.
+
+#### `playlistId`
+* *Type:* string | number
```

Comment:
should we also support these as objects with `limit`, `id`, etc?

Reply:
What I'm getting at is we don't support `limit`, etc. in the embed code (except `catalogSearch` and `catalogSequence`).